pkg/nuget/packages_props/parse.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"encoding/xml"
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+	
+
+
+type propsPackageEntry struct{
+	Version         string   `xml:"Version,attr"`
+	UpdatePackageName     string   `xml:"Update,attr"`
+	IncludePackageName     string   `xml:"Include,attr"`
+}
+
+type propsPackageReferenceEntry struct {
+	XMLName         xml.Name `xml:"PackageReference"`
+	propsPackageEntry
+}
+
+type propsPackageVersionEntry struct {
+	XMLName         xml.Name `xml:"PackageVersion"`
+	propsPackageEntry
+}
+
+type propsItemGroup struct {
+	XMLName  xml.Name              `xml:"ItemGroup"`
+	ReferencePackages []propsPackageReferenceEntry `xml:"PackageReference"`
+	VersionPackages []propsPackageVersionEntry `xml:"PackageVersion"`
```

Comment:
Looks like old and new `*.packages.props` files only use `ReferencePackages`.
Can you add link to docs/example for `VersionPackages` and add this case in tests?

Reply:
Sure.
Packages props example:
https://github.com/dotnet/msbuild/blob/main/eng/Packages.props
Link from dotnet repo where these packages are parsed. Both Package Reference and Package Version are considered:
https://github.com/dotnet/roslyn-tools/blob/380ffede53738d15ed1384e0f52dd8a3d937d38b/src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs#L495

Reply:
An example of Directory.packages.props using packageVersion
https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-package-versions

Reply:
Okay, thanks.
Can you add link to this example in comment, please.

Hm.. look like version can use variables, but we can't resolve them, right? https://github.com/dotnet/msbuild/blob/31c4d335325e858a3c4dba66d921d0e31bdee5ff/eng/Packages.props#L11
Maybe we need to remove them from result or consider finding these values from another file and merging result in Trivy (we use this logic to find licenses from nuspec files when parsing NuGet lock and config files - https://github.com/aquasecurity/trivy/blob/d0d956fdc1aa71ddf7f806888ea2b64d8441bfa1/pkg/fanal/analyzer/language/dotnet/nuget/nuget.go#L82)

Also i think it is better to use same name for fields from structure and xml tag
I mean renaming: ReferencePackages -> PackageReference and VersionPackages -> PackageVersion